processor/stream/benchmark_test.go

```diff
@@ -49,7 +48,7 @@ func benchmarkStreamProcessor(b *testing.B, processor *Processor, files []string
 		return nil
 	}
 	//ensure to not hit rate limit as blocking wait would be measured otherwise
-	rl := rate.NewLimiter(rate.Limit(math.MaxFloat64-1), math.MaxInt32)
+	// rl := rate.NewLimiter(rate.Limit(math.MaxFloat64-1), math.MaxInt32)
```

Comment:
you probably didn't mean to commit this?

Reply:
no 🤦‍♀️